### PR TITLE
feat: Support customize buy product page

### PR DESCRIPTION
**Pull request by ljl66-66**
fix: [#3325](https://github.com/casdoor/casdoor/issues/3325)

### CODE REVIEW

**Comment by hsluoyz** [comment]
@dacongda plz review

**Review by dacongda — COMMENTED** [review]
can you provide a demo video?

File: web/src/locales/zh/data.json [hunk 1]
```diff
@@ -583,6 +583,12 @@
     "Account items": "个人页设置项",
     "Account items - Tooltip": "用户的个人设置页面中可配置的选项",
     "All": "全部",
+    "Buy Product Page CSS": "购买商品页面CSS",
```
[the remark this answers is not in the document]
Comment (dacongda) [reply web/src/locales/zh/data.json:586]: and `Buy Product Page CSS` should be` Buy product page CSS`

[the remark this answers is not in the document]
Comment (hsluoyz) [reply web/src/locales/zh/data.json:586]: @ljl66-66 see: https://casdoor.org/docs/contributing/#how-to-update-i18n-data

**Comment by ljl66-66** [comment]
This is a demo video: 
https://github.com/user-attachments/assets/3bb3ff33-0a47-40d4-9673-bea4f8b2812f

**Comment by hsluoyz** [comment]
@dacongda plz review

File: web/src/ProductBuyPage.js [hunk 2]
```diff
@@ -37,6 +71,7 @@ class ProductBuyPage extends React.Component {
       plan: null,
       isPlacingOrder: false,
       customPrice: 0,
+      organization: null,
```
Comment (dacongda) [inline web/src/ProductBuyPage.js:74]: is organization useful?

**Pull request closed without merging**